### PR TITLE
Populate pull requests

### PR DESCRIPTION
**Pull request by michaelpaul**
## Summary

Selectively copy generated code into the respective library's pull request.

### CODE REVIEW

File: java/build.gradle [hunk 1]
```diff
@@ -1,3 +1,81 @@
...
+
+    library.set("jersey3")
+    modelPackage.set(modelNs.replace('/', '.'))
+    apiPackage.set("com.adyen.service.${svcId}")
+    apiNameSuffix.set('Api')
+    additionalProperties.putAll([
+            'dateLibrary'    : 'java8',
+            'openApiNullable': 'false',
+    ])
+    // @TODO check if this is still used per service (or on models)
```
Comment (wboereboom) [inline java/build.gradle:27]: Should this TODO still be in here?

File: java/build.gradle [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+
+project.ext {
+    generator = 'java'
+}
+
+def services = project.ext.services as List<Service>
+def smallServices = project.ext.smallServices as List<Service>
+
+tasks.withType(GenerateTask).configureEach {
+    def svcId = project.ext.serviceName.toLowerCase()
```
Comment (wboereboom) [inline java/build.gradle:16]:
```suggestion
    def serviceId = project.ext.serviceName.toLowerCase()
```

File: java/build.gradle [hunk 3]
```diff
@@ -1,3 +1,81 @@
...
+    def modelNs = "com.adyen.model.${svcId}"
```
Comment (wboereboom) [inline java/build.gradle:17]:
```suggestion
    def modelNamespace = "com.adyen.model.${svcId}"
```

File: python/build.gradle [hunk 4]
```diff
@@ -1,8 +1,82 @@
...
+    removeTags = false
+}
+
+List<Service> services = project.ext.services
+services.find { it.id == 'payment' }.small = false
+
+// Service renaming
+Map<String, String> serviceNaming = project.ext.serviceNamingCamel
+serviceNaming.putAll([
+        'binlookup'            : 'binlookup',
```
Comment (wboereboom) [inline python/build.gradle:19]: Both are called `binlookup`. What is being mapped here?

Comment (michaelpaul) [reply python/build.gradle:19]: The key it's the service ID and the value it's new name. The original would have a different case, it's [BinLookup](https://github.com/Adyen/adyen-dotnet-api-library/tree/main/Adyen/Model/BinLookup).

File: java/build.gradle (hunk 1 above)
Comment (michaelpaul) [reply java/build.gradle]: Double checked and it's unused.

**Review by wboereboom — APPROVED** [review]

**Review by AlexandrosMor — APPROVED** [review]

**Pull request merged**